homeassistant/components/homekit/accessories.py

```diff
@@ -9,13 +9,14 @@
 from pyhap.const import CATEGORY_OTHER
 
 from homeassistant.const import (
-    __version__, ATTR_BATTERY_CHARGING, ATTR_BATTERY_LEVEL)
+    __version__, ATTR_BATTERY_CHARGING, ATTR_BATTERY_LEVEL, ATTR_ENTITY_ID)
 from homeassistant.core import callback as ha_callback
 from homeassistant.core import split_entity_id
 from homeassistant.helpers.event import (
     async_track_state_change, track_point_in_utc_time)
 from homeassistant.util import dt as dt_util
 
+
```

Comment:
Wanted to be thorough. :-) Yeah, figured I'll clean it up once we have a final approach. :-)

Reply:
Quick tip: Don't know if you have seen this already: https://developers.home-assistant.io/docs/en/development_testing.html#testing-outside-of-tox

Especially the part about `script/lint`. I run it all the time when I'm working on something new.

Reply:
Yeah, right after I did the commit and put it in the PR and then saw the check items come back I knew I forgot something. :-)
Figured that instead of going back and fixing it right then to first get your initial feedback and then fix extra lines etc. together with changes based on your feedback. :-)

Ran it all this time around, so should come back good. :-)